github/pulls_reviews_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestPullRequestsService_ListReviews(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/pulls/1/reviews", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypePullRequestReview)
+		fmt.Fprint(w, `[{"id":1},{"id":2}]`)
+	})
+
+	reviews, _, err := client.PullRequests.ListReviews("o", "r", 1)
+	if err != nil {
+		t.Errorf("PullRequests.ListComments returned error: %v", err)
+	}
+
+	want := []*PullRequestReview{{ID: Int(1)}, {ID: Int(2)}}
```

Comment:
Minor thought, but I'd make it more visible there are 2 reviews here by using multiple lines, like so:

```Go
want := []*PullRequestReview{
	{ID: Int(1)},
	{ID: Int(2)},
}
```

For example, see [here](https://github.com/google/go-github/blob/466070b0580728e63bd1a415e0019639e55d7148/github/pulls_test.go#L305-L324).